tests/Application/config/sylius/grids/book.php

```diff
@@ -31,6 +32,11 @@
             null,
             ['author.nationality'],
         ))
+        ->addFilter(AttributeNationalityFilter::create(
```

Comment:
How is it tested? Does it override nationality filter and is covered by this test: https://github.com/Sylius/SyliusGridBundle/blob/f500a89f5ea35fd675012c30e55d0def0a751cc3/src/Bundle/Tests/Functional/GridUiTest.php#L235-L245 or is not covered by test and we don't know if it works?

Reply:
I think it's covered in https://github.com/Sylius/SyliusGridBundle/blob/f500a89f5ea35fd675012c30e55d0def0a751cc3/tests/Application/src/Filter/AttributeNationalityFilter.php and https://github.com/Sylius/SyliusGridBundle/blob/f500a89f5ea35fd675012c30e55d0def0a751cc3/tests/Application/src/Grid/Builder/AttributeNationalityFilter.php